components/expenses/Expense.js

```diff
@@ -461,7 +461,11 @@ class Expense extends React.Component {
                       />
                     )}
                     {canPay && (
-                      <StyledButton mr={2} onClick={() => this.setState({ showUnapproveModal: true })}>
+                      <StyledButton
+                        mr={2}
+                        buttonStyle="dark"
```

Comment:
We should keep the default state for this one, unless there's a good rational for using a dark style